modules/tinymce/src/themes/silver/test/ts/browser/skin/OxideColorSwatchMenuTest.ts

```diff
@@ -164,4 +166,15 @@ describe('browser.tinymce.themes.silver.skin.OxideColorSwatchMenuTest', () => {
     assertFocusIsOnColor('rgb(224, 62, 45)');
     closeForecolorMenu();
   });
+
+  it('TINY-9342: selected color is successfully marked even in a tree', async () => {
+    const editor = hook.editor();
+    editor.setContent('<pre><span style="background-color: rgb(224, 62, 45);"><b>red</b></span></pre>');
+
+    TinySelections.setSelection(editor, [ 0, 0, 0, 0 ], 0, [ 0, 0, 0, 0 ], 1, true);
```

Comment:
Not sure it needs to be a expanded selection. Shorter code.

```suggestion
    TinySelections.setCursor(editor, [ 0, 0, 0, 0 ], 1, true);
```